css-three-column-layout/styles.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  background-color: rgb(25, 150, 218);
+}
+
+#container {
+  margin: auto;
+  width: 900px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  width: 100%;
+}
+
+img {
+  width: 40%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+div {
+  font-family: 'Lato', sans-serif;
+  color: white;
+  margin: 10px;
+}
+
+p {
+  text-align: center;
+  font-size: 20px;
+  width: 275px;
```

Comment:
This shouldn't be necessary, you should just use some `padding` so the width remains unaltered

Reply:
This still needs to be addressed